### ISSUE TITLE
Can't get it working

### ISSUE THREAD

**Issue by ErkoKnoll** [stackblitz/monaco-auto-import#1]
Instructions are quite clear, but I still can't get it to work.
Setup:
```
const completor = new AutoImport({
            monaco,
            editor,
        })
        completor.imports.saveFile({
            path: 'testPackage',
            imports: [
                {
                    name: 'QQQ',
                    type: 'interface'
                }, {
                    name: 'www',
                    type: 'function'
                }
            ]
        })
        completor.attachCommands()
```

But when trying to auto import `www` I'm getting nothing:
<img width="470" alt="Screenshot 2020-03-11 at 15 25 03" src="https://user-images.githubusercontent.com/9930079/76421854-f853fd00-63ac-11ea-958a-578f4045e4d8.png">

I'm using Monaco version 0.20.0 and I'm also noticing an error being thrown from Monaco side when trying to invoke suggestions which might be the cause:
```
TypeError: Cannot read property 'filter' of undefined
    at codeAction.js:136
    at step (codeAction.js:49)
    at Object.next (codeAction.js:30)
    at fulfilled (codeAction.js:21)
    at errors.js:12
```
<img width="861" alt="Screenshot 2020-03-11 at 15 31 07" src="https://user-images.githubusercontent.com/9930079/76422112-5d0f5780-63ad-11ea-8a4c-3f72b834089d.png">

**Comment by rizrmd** [comment]
Yeah me too! maybe api for generating import are changed on monaco-editor 0.20.0

**Comment by ErkoKnoll** [comment]
@rizkyramadhan APIs have definitely changed. It was a while ago but I checked this implementation and recreated what I needed for newer Monaco version, it was relatively easy.

**Comment by rizrmd** [comment]
> @rizkyramadhan APIs have definitely changed. It was a while ago but I checked this implementation and recreated what I needed for newer Monaco version, it was relatively easy.

Yup, i just done it. Thanks to this repo, my code is so messy that I think it's not worth for me to share it.
